config/cp4waiops/install-aimgr/Chart.yaml

```diff
@@ -16,9 +16,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1
+version: 0.0.2
```

Comment:
This needs to be `3.4.0` per https://github.com/IBM/cp4waiops-gitops/issues/177#issuecomment-1171883532

The same applies to `version` defined in `install-emgr/Chart.yaml`.